src/main/java/com/org/checkr/controller/CandidatesController.java

```diff
@@ -43,6 +43,9 @@ public ResponseEntity<CompleteCandidateInfoDTO> createCandidate(@RequestBody Cre
     @PatchMapping("/{id}")
     public ResponseEntity<CompleteCandidateInfoDTO> engageWithCandidate(@PathVariable Long id) {
         Candidate candidate = candidateService.engageWithCandidate(id);
+        if (candidate == null) {
+            return new ResponseEntity<>(null, HttpStatus.OK);
```

Comment:
status should be not found instead of OK

Reply:
corrected the response code